### DISCUSSION TITLE
如何调整缓存设置或关闭,每当发布新文章时,旧的文章都需要刷新3次才能够刷新出新的文章目录

### DISCUSSION THREAD

**Discussion by Tamrac-web** [tangly1024/NotionNext#2656] [Q&A]
发布新文章后,目录需要刷新2-3次才会更新目录,此时点击另一篇旧的文章,目录仍是旧的目录,需要在旧的文章中刷新才会更新到最新的目录,更换主题也是,点击没访问过的旧文章,仍然是旧的主题,需要刷新才会更新到最新的主题.

**Comment by LooseLi** [comment]
根据tang老师 [NotionNext帮助手册](https://docs.tangly1024.com/article/cache-of-notion-next) 上的描述，这应该是站点的缓存机制，也是为了用户的使用体验😃 所以目前应该不支持自定义设置
